### ISSUE TITLE
User can focus in on current todo

### ISSUE THREAD

**Issue by avidrucker** [avidrucker/autofocus-exp#50]
ala the zen design of https://momentumdash.com/

Also called "focus mode", the user sees simply the current most want to do item, and a prompt (or button) to leave the screen / progress forward.

Note: This is not a "screen" per se, but simply a clearing of the console, a display of the current in-progress todo item, and a prompt to hit the enter key to answer the question, "Is there further work left on this todo item? (y/n, or, hit 'q' then 'enter' to quit without saving progress) " where the next "screen" (display) is the return to main menu (prompt).

To sum it up:
- [x] 1. User sees the console cleared
- [x] 2. User sees current in-progress todo item (the cmwtd)
- [ ] 3. User can hit any key to indicate they are "done working for now"
- [ ] 4A. User can answer whether there is work left to do in the future (y/n/quit)
- [ ] 4B. User can quit to indicate that the cmwtd shouldn't be marked as complete
- [ ] 5. User is returned to the main menu

**Comment by avidrucker** [comment]
Question for Architect: What is the specific text of the prompt to progress app state forward ? What is the resultant app state (screen, and logic flow/change) ?

**Comment by avidrucker** [comment]
- [ ] Architect assigns associated tests for
   - [ ] Command line app
   - [ ] Desktop electron app
   - [ ] Web app
